modules/search/client/views/search-sidebar-results.client.view.html

```diff
@@ -97,6 +97,8 @@ <h4>
                   'btn-offer-hosting-maybe': search.offer.status === 'maybe'
                 }"
           ></span>
+          <div>Online {{search.offer.user.lastSeen}}</div>
```

Comment:
Could we put this in some shared place and re-use?

https://github.com/Trustroots/trustroots/blob/061847721413977c17c9623b23c68ed5351a8a0b/modules/users/client/components/ProfileViewBasics.js#L31-L36

Lemme know if I can help to show how to use React components in Angular templates.